ttnn/cpp/ttnn/operations/conv2d/conv2d_pybind.hpp

```diff
@@ -55,6 +55,65 @@ void py_module(py::module& module) {
         py::arg("bias_tensor") = std::nullopt,
         py::arg("conv_config") = std::nullopt);
 
+    module.def(
+        "optimized_conv",
+        &optimized_conv,
+        py::arg().noconvert(),
+        py::arg().noconvert(),
+        py::arg("bias").noconvert() = std::nullopt,
+        py::arg("conv_reader_indices").noconvert() = std::nullopt,
+        py::arg().noconvert(),
+        py::arg().noconvert(),
+        py::arg().noconvert(),
+        py::arg().noconvert(),
+        py::arg().noconvert(),
+        py::arg().noconvert(),
+        py::arg().noconvert(),
+        py::arg().noconvert(),
+        py::arg().noconvert() = 0,
+        py::arg("output_mem_config").noconvert() = std::nullopt,
+        py::arg("output_dtype").noconvert() = std::nullopt,
+        py::arg("input_tensor_shape").noconvert() = std::nullopt,
+        py::arg("use_shallow_conv_variant").noconvert() = false,
+        py::arg("transpose_mcast").noconvert() = true,
+        py::arg("compute_kernel_config").noconvert() = std::nullopt,
```

Comment:
where is documentation on these params?

Reply:
[This](https://tenstorrent.sharepoint.com/:p:/r/sites/Jasmina/Shared%20Documents/tt-metal/ttDNN/convs/[Conv_implementation.pptx](https://tenstorrent.sharepoint.com/:p:/r/sites/Jasmina/Shared%20Documents/tt-metal/ttDNN/convs/Conv_implementation.pptx?d=w783675a2be89434db847873bb18eed42&csf=1&web=1&e=62dwc9&nav=eyJzSWQiOjUzNywiY0lkIjozMzc1ODI2MjA2fQ)?d=w783675a2be89434db847873bb18eed42&csf=1&web=1&e=62dwc9&nav=eyJzSWQiOjUzNywiY0lkIjozMzc1ODI2MjA2fQ) is the closest we have to documentation.

Reply:
I don't believe the doc is in the code. @sankarmanoj-tt we can copy paste the arg docs from the conv slides